src/dygraph.js

```diff
@@ -2533,16 +2533,22 @@ Dygraph.prototype.computeYAxisRanges_ = function(extremes) {
     if (!ypadCompat) {
       // When using yRangePad, adjust the upper/lower bounds to add
       // padding unless the user has zoomed/panned the Y axis range.
+
+      y0 = axis.computedValueRange[0];
+      y1 = axis.computedValueRange[1];
+
+      // special case #781: if we have no sense of scale, center on the sole value.
+      if (y0 === y1) {
+          y0 -= 0.5;
```

Comment:
For a log scale, this could drop y below zero, which will produce `NaN`s. In the `logscale` branch of the conditional on line 2546, you can halve y0 and double y1 instead.